coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/functions/StaticHiveFunctionRegistry.java

```diff
@@ -667,6 +667,8 @@ public boolean isOptional(int i) {
         family(SqlTypeFamily.STRING, SqlTypeFamily.ANY, SqlTypeFamily.TIMESTAMP));
     createAddUserDefinedFunction("com.linkedin.policy.decoration.udfs.RedactFieldIf", ARG1,
         family(SqlTypeFamily.BOOLEAN, SqlTypeFamily.ANY, SqlTypeFamily.STRING, SqlTypeFamily.ANY));
+    createAddUserDefinedFunction("li_groot_cast_nullability", new OrdinalReturnTypeInferenceV2(1),
+        family(SqlTypeFamily.ANY, SqlTypeFamily.ANY));
```

Comment:
Why do we need 2 input parameters for this function? Asking because in the unit test, two parameters are same.

Reply:
`li_groot_cast_nullability` will take 2 args which may / maynot be the same and can have any type. The idea is to cast the nullability of ARG2 onto ARG1 where ARG1 can be same as ARG2 or a processed version of ARG2 such as udf(ARG2).